.github/workflows/crowdin_wf.yml

```
@@ -0,0 +1,57 @@
+# CrowdIn workflow for SMF
+
+name: CrowdIn
+
+# Controls when the action will run. 
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  # Auto pushes when a release is tagged
+  release:
+    types: [ published ]
+    branches: [ release-2.1 ]
+
+# Only one job, the crowdin sync
+jobs:
+  crowdin-sync:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Checkout the source, then invoke the GitHub CrowdIn Action
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: crowdin-action
+      # You need to pin to a specific version for some reason...
+      uses: crowdin/github-action@1.0.19
+      with:
+        # Push sources (the english files) to CrowdIn; do not push translations (non-english) in either direction
+        upload_sources: true
+        upload_translations: false
+        download_translations: false
+        # The crowdin_branch_name is the name of the highest level folder in your "Files" tab in CrowdIn
+        # At the moment, it really does not like embedded spaces (2/2021)
+        crowdin_branch_name: 'SMF_2-1'
+        # The rest of the "with" items below are only needed when you want translations back into GH...
+        push_translations: true
+        # Commit message for download translations
+        commit_message: 'New Crowdin translations by Github Action'
+        # To download translations to the specified version branch
+        localization_branch_name: 'l10n_crowdin_action'
+        # Create pull request after pushing to branch
+        create_pull_request: true
+        # The title of the new pull request
+        pull_request_title: 'New Crowdin translations by Github Action'
+        # To add labels for created pull request
+        pull_request_labels: 'Language'
+
+      env:
+        # Use GitHub Secrets here, so sensitive stuff isn't in the publicly visible workflow
+        # Note that the CrowdIn API token will accept either a user's personal CrowdIn token
+        # or an "API" token, generated via the CrowdIn console client using the "crowdin init" command.
+        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+        CROWDIN_PROJECT_ID: ${{ secrets.CI_PROJECT }}
+        CROWDIN_API_TOKEN: ${{ secrets.CI_API_TOKEN }}
```

Your tokens should be more descriptive.
```suggestion
        CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
        CROWDIN_API_TOKEN: ${{ secrets.CROWDIN_API_TOKEN }}
```

---

Good input.